Lib/test/test_asyncgen.py

```diff
@@ -322,6 +322,7 @@ def test_async_gen_api_01(self):
 class AsyncGenAsyncioTest(unittest.TestCase):
 
     def setUp(self):
+        self.addCleanup(asyncio.set_event_loop, asyncio.get_event_loop())
```

Comment:
Why do we need this?

Reply:
Because the tests were wiping out the event loop and were triggering the mutated environment check for the event loop.